contributing-guides/style-guide.md

```diff
@@ -6,27 +6,27 @@ This page lists specific formatting instructions for `tldr` pages.
 
 The basic format of each page should match the following template:
 
-```
+```md
 # command-name
 
 > Short, snappy description.
 > Preferably one line; two are acceptable if necessary.
-> More information: <https://example.com>.
+> More information: <https://example.com/command-name/man/page>.
```

Comment:
```suggestion
> More information: <https://example.com>.
```
Often, if possible, links are to resources relating, instead of a manual page, and your suggestion promotes it which I personally do not like.

Reply:
I've replaced it with: `https://example.com/command-name/help/page`. And added a note:

```
> :bulb: Help page is an any documentation page here, not just man page.
```

Reply:
Generally we prefer, in order or preference:

1. Specific official documentation on the command-line program in question
2. The project's homepage
3. A link to a man page